weave-js/src/components/PagePanelComponents/Home/HomeLeftNav.tsx

```diff
@@ -71,19 +73,21 @@ type LeftNavItemProps = {
   icon: React.FC;
   label: string;
   active?: boolean;
+  to: string;
   onClick?: () => void;
```

Comment:
Does it ever make sense to have `onClick` if `to` is set?

Reply:
Yes, in some cases we clear state in the onClick that isn't an issue if you are loading the page fresh.